sfaira/data/dataloaders/loaders/d10_1038_s41586_019_1289_x/homosapiens_brain_2019_10x3v2_velasco_001.py

```diff
@@ -0,0 +1,30 @@
+import anndata as ad
+import os
+import pandas as pd
+import numpy as np
+import scipy
+
+
+def load(data_dir, sample_fn, **kwargs):
+    fn = os.path.join(data_dir, sample_fn)
+    fn_meta = os.path.join(data_dir, "meta_combined.txt")
+
+    expr_df = pd.read_csv(fn, sep="\t", compression='gzip', index_col=0)
+    expr_df = expr_df.apply(np.expm1)
+    expr_df = expr_df.T
+
+    metadata_df = pd.read_csv(fn_meta, sep="\t", index_col=0, header=0)
+    metadata_df = metadata_df.tail(-1)
+
```

Comment:
Let's add an extra line to specifically store organoid age in days as well:
```python
metadata_df["organoid_age_days"] = pd.Series([{"3mon": "90", "6mon": "180"}[i] for i in metadata_df["Batch"].str.split("_").str[1]], dtype="category")
```

Reply:
done